src/main/java/com/microsoft/aad/msal4j/JsonHelper.java

```diff
@@ -29,4 +31,73 @@
             throw new MsalClientException(e);
         }
     }
+
+    /**
+     * Merges given JSON strings into one Jackson JSONNode object, which is returned as a String
+     */
+    static String mergeJSONString(String mainJsonString, String addJsonString) {
+        JsonNode mainJson, addJson;
+        try {
+            mainJson = mapper.readTree(mainJsonString);
+            addJson = mapper.readTree(addJsonString);
+        } catch (JsonProcessingException e) {
+            throw new MsalClientException(e.getMessage(), AuthenticationErrorCode.INVALID_JSON);
+        }
+
+        mergeJSONNode(mainJson, addJson);
+
+        return mainJson != null ? mainJson.toString() : null;
```

Comment:
I think we should throw if null here, instead of returning null - there is no scenario I can think of where you would call `mergeJSONString` and have null be a valid return value. This might save us from getting a null ref error somewhere else.